### PR TITLE
Linda Norbergs portfolio w4

### PR DESCRIPTION
**Pull request by lindanorberg**
I did not manage to get the whole project done. Theese are known problems 
- tabindex is not implemented on the whole page
- profile picture is sometimes hiding the hero-text
- heroimage is not scaling good enough 
- on desktop profie picture and hero text is positioned too far apart
- the skills-section needs more work
- I need to add more texts and links
- it lacks the tags in the featured projects area 

But I think the screen reader is working ok.... 🤣

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="./reset-style.css">
```
Comment (NinaBerggren) [inline index.html:8]: Great thet you used reset-style.css

File: index.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    <link rel="stylesheet" href="./style.css">
+
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+
+    <title>Linda Norberg's portfolio</title>
+</head>
+<body>
+    <main class="outer-wrapper">
```
Comment (NinaBerggren) [inline index.html:16]: excellent start to use outer-wrapper and row:28 inner-wrapper for easy styling the hole webpage
Question; should header and footer be included in this styling elements?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+            <section class="skills white-section">
+                <h2 class="section-title">SKILLS</h2>
+                <div class="column-1">
+                    <div>
+                        <h3 class="section-sub-title">Code</h3>
+                        <p>HTML5</p>
+                        <p>CSS3</p>
+                        <p>JavaScript ES6</p>
+                    </div>
```
Comment (NinaBerggren) [inline index.html]: you can use an unordered list with div-named containers for easy styling.
ex
`<div class="skills-container>`
      `  <div class="skills-container-item" tabindex="0">`
                    `<h4 class="skills-header">CODE</h4>`
                    `<ul>`
                       ` <li>HTML5</li>`
                        `<li>CSS3</li>`
                       ` <li>JavaScript ES6</li>    `       
                    `</ul>`
             `</div>`
            `<div class="skills-container-item" tabindex="0">`
                    `<h4 class="skills-header">Toolbox</h4>`
                    `<ul>`
                        `<li>VSC</li>`
                        `<li>Github</li>`
                        `<li>Figma</li> `
                        `<li>Slack</li>`
                    `</ul>`
             `</div>`
etc for each column
`</div>`
sorry for the compact code snippet, don't know how to fixit for now

File: style.css [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+/* tablet second  */
+@media screen and (min-width: 668px) {
+    .inner-wrapper {
+        max-width: 800px;
+    }
+
+    .profile-pic {
+        max-height: 200px;
+        /*Do something about covering text, overlap underlap. maybe change to static instead of absolut...*/ 
+    }
```
Comment (NinaBerggren) [inline style.css]: To fix the problem with hero-text and profile-pic on top of each other,
in @media query, style both profile-pic and hero-text, change the position on both. Perhaps change the font size for the different "devices"

**Review by NinaBerggren — COMMENTED** [review]
Smart to use different class name for each ”section” for easy styling of background color.

Tip for easy reading and finding your code sections,
divide the code with comments/headings use :
<!—comment—!> in HTMS
 /*  comment */ in CSS 
// comment    in JS

I know you're not finished with this project,  but you are heading in the right direction.
Keep up the good work, I know you can do it!

**Review by NinaBerggren — COMMENTED** [review]
pls let me know if there is anything else you need help with finishing this project =)

**Comment by lindanorberg** [comment]
Thank you so much for your comments Nina! They are helpful! I will get back to this project, when my feelings for it has chilled down a notch 🤣

**Pull request closed without merging**